FuelSDK/objects.py

```diff
@@ -87,15 +87,35 @@ def __init__(self):
 ##  wrap an Exact Target List and List Subscriber
 ##
 ########
+class ET_Group(ET_CUDSupport):
+    def __init__(self):
+        super(ET_Group, self).__init__()
+        self.obj_type = 'Group'
+
+class ET_Send(ET_CUDSupport):
+    def __init__(self):
+        super(ET_Send, self).__init__()
+        self.obj_type = 'Send'
+
+class ET_ListSend(ET_CUDSupport):
+    def __init__(self):
+        super(ET_ListSend, self).__init__()
+        self.obj_type = 'ListSend'
+
 class ET_List(ET_CUDSupport):
     def __init__(self):
         super(ET_List, self).__init__()
         self.obj_type = 'List'
-    
+
 class ET_List_Subscriber(ET_GetSupport):
     def __init__(self):
         super(ET_List_Subscriber, self).__init__()
-        self.obj_type = 'ListSubscriber'    
+        self.obj_type = 'ListSubscriber'
+
+class ET_SubscriberList(ET_GetSupport):
+    def __init__(self):
+        super(ET_Subscriber_List, self).__init__()
```

Comment:
This is a syntax error. The first argument of `super` must be `ET_SubscriberList` which matches the class name not `ET_Subscriber_List`.